content/ko/docs/tutorials/stateful-application/basic-stateful-set.md

```diff
@@ -0,0 +1,897 @@
+
+for i in 0 1; do kubectl exec -it web-$i -- curl localhost; done
+web-0
+web-1
+```
+
+{{< note >}}
+위에 curl 명령어로 403 Forbidden 응답 아닌 것을 보려면 `volumeMounts`로 마운트된 디렉터리의 퍼미션을 고쳐야 한다([hostPath 볼륨을 사용할 때에 버그](https://github.com/kubernetes/kubernetes/issues/2630)로 인함).
+
+```shell
+for i in 0 1; do kubectl exec web-$i -- chmod 755 /usr/share/nginx/html; done
+```
+
+위에 curl 명령을 재시도하기 전에
+{{< /note >}}
+
+첫째 터미널에서 스테이트풀셋의 파드를 감시하자.
+
+```shell
+kubectl get pod -w -l app=nginx
+```
+
+두번째 터미널에서 스테이트풀셋의 모든 파드를 삭제하자.
+
+```shell
+kubectl delete pod -l app=nginx
+pod "web-0" deleted
+pod "web-1" deleted
+```
+
+첫번째 터미널에서 실행 중인 `kubectl get`명령어의 출력을 확인하고, 모든 파드가 Running과 Ready 상태로 전환될때까지 기다리자.
+
+```shell
+kubectl get pod -w -l app=nginx
+NAME      READY     STATUS              RESTARTS   AGE
+web-0     0/1       ContainerCreating   0          0s
+NAME      READY     STATUS    RESTARTS   AGE
+web-0     1/1       Running   0          2s
+web-1     0/1       Pending   0         0s
+web-1     0/1       Pending   0         0s
+web-1     0/1       ContainerCreating   0         0s
+web-1     1/1       Running   0         34s
+```
+
+웹서버에서 자신의 호스트네임을 계속 제공하는지 확인하자.
+
+```
+for i in 0 1; do kubectl exec -it web-$i -- curl localhost; done
+web-0
+web-1
+```
+
+비록 `web-0`과 `web-1`이 재스캐쥴링되어도 계속해서 자신의 호스트네임을 제공하는데 이는 각 퍼시스턴트볼륨요청에 연관된 퍼시스턴트볼륨이 해당 `volumeMounts`로 재마운트되기 때문이다. `web-0`과 `web-1`의 스케쥴링에 관계없이 각각의 퍼시스턴트볼륨은 적절하게 마운트된다.
+
+## StatefulSet 스케일링
+
+스테이트풀셋을 스케일링하는 것은 리플리카 개수를 늘리거나 줄이는 것을 의미한다. 이것은 `replicas` 필드를 갱신하여 이뤄진다. [`kubectl scale`](/docs/reference/generated/kubectl/kubectl-commands/#scale)이나 [`kubectl patch`](/docs/reference/generated/kubectl/kubectl-commands/#patch)을 이용해서 스테이트풀셋을 스케일링할 수 있다.
+
+### 스케일 업
+
+한 터미널창에서 스테이트풀셋의 파드를 감시하자.
+
+```shell
+kubectl get pods -w -l app=nginx
+```
+
+다른 터미널창에서 `kubectl scale`을 이용하여 리플리카 개수를 5로 스케일링하자.
+
+```shell
+kubectl scale sts web --replicas=5
+statefulset.apps/web scaled
+```
+
+첫번째 터미널에서 실행 중인 `kubectl get`명령어의 출력을 확인하고, 3개의 추가 파드가 Running과 Ready 상태로 전환될때까지 기다리자.
+
+```shell
+kubectl get pods -w -l app=nginx
+NAME      READY     STATUS    RESTARTS   AGE
+web-0     1/1       Running   0          2h
+web-1     1/1       Running   0          2h
+NAME      READY     STATUS    RESTARTS   AGE
+web-2     0/1       Pending   0          0s
+web-2     0/1       Pending   0         0s
+web-2     0/1       ContainerCreating   0         0s
+web-2     1/1       Running   0         19s
+web-3     0/1       Pending   0         0s
+web-3     0/1       Pending   0         0s
+web-3     0/1       ContainerCreating   0         0s
+web-3     1/1       Running   0         18s
+web-4     0/1       Pending   0         0s
+web-4     0/1       Pending   0         0s
+web-4     0/1       ContainerCreating   0         0s
+web-4     1/1       Running   0         19s
+```
+
+스테이트풀셋 컨트롤러는 리플리카개수를 스케일링한다. [스테이트풀셋 생성](#ordered-pod-creation)으로 스테이트풀셋 컨트롤러는 각 팟을 순차적으로 각 순번에 따라 생성하고 후속 파드 시작 전에 이전 파드가 Running과 Ready가 될때까지 기다린다.
+
+### 스케일 다운
```

Comment:
```suggestion
### 스케일 다운 {#scaling-down}
```

so that https://github.com/kubernetes/website/pull/13073/files#diff-2efc7d17d185214a0b149ccf64139586R742 works OK.